modules/solid_mechanics/doc/content/source/materials/MultiPhaseStressMaterial.md

```diff
@@ -1 +1,27 @@
-!template load file=stubs/moose_object.md.template name=MultiPhaseStressMaterial syntax=/Materials/MultiPhaseStressMaterial
\ No newline at end of file
+# MultiPhaseStressMaterial
+
+!syntax description /Materials/MultiPhaseStressMaterial
+
+The `MultiPhaseStressMaterial` is used in coupled phase field - solid mechanics calculations to compute
+the mechanical deformation of multi-phase regions.
+
+The switching (mixing) coefficients which blends each phase stresses together must be provided
```

Comment:
Should this be `each phase stress together` since each usually denotes singular nouns? In "TwoPhaseStressMaterial" there is a similar sentence, "blends the two phase stresses together". Should `each` be replaced with `the`?